posthog-node/src/posthog-node.ts

```diff
@@ -338,6 +338,59 @@ export class PostHog implements PostHogNodeV1 {
     return response
   }
 
+  async getAllPayloads(
```

Comment:
Thinking about this a bit more, for the final interface as well, I don't think anyone would want _just_ payloads, so we should turn this into `getAllFlagsAndPayloads` as well. Thoughts?

Imagine someone bootstrapping, ideally they'll just want one request max to get everything they need to pass to the frontend, as this will be very time sensitive.

Reply:
and then we can combine the above and this to use `getAllflagsAndPayloads` only, while `getAllFlags` just returns the flags from `getAllflagsAndPayloads`